indico/htdocs/js/custom/dropzone.js

```diff
@@ -39,7 +39,7 @@
                 var self = this;
                 var file = $dz.data('value');
 
-                if (file) {
+                if (file && typeof file === 'object') {
```

Comment:
I think now that I fixed the problem with the form embedded dropzone, this is not necessary anymore